frappe/translate.py

```diff
@@ -808,19 +808,25 @@ def write_csv_file(path, app_messages, lang_dict):
 				w.writerow([message, translated_string, context])
 
 
-def get_untranslated(lang, untranslated_file, get_all=False):
+def get_untranslated(lang, untranslated_file, get_all=False, app='all'):
```

Comment:
Probably best to keep a default that has lesser chances of collision - an app can exist with the name `all` too.

```suggestion
def get_untranslated(lang, untranslated_file, get_all=False, app='_ALL_APPS'):
```